packages/babel-plugin-jsx-dom-expressions/src/shared/utils.js

```diff
@@ -224,14 +224,6 @@ export function toEventName(name) {
   return name.slice(2).toLowerCase();
```

Comment:
Having just `name.slice(2)` here would be nice. Then event delegation will work even for custom events regardless of the name right?

ALong with that, what about a way to configure events, f.e. `onMyEvent={[e => console.log(e), {bubbles: false, delegated: true}]}` or `onotherEvent={{handleEvent: e => console.log(e), bubbles: false, delegated: true}}`